packages/aft/test/version_bump/data/snapshots/multi_package_update.diff

```diff
@@ -60,13 +56,16 @@ diff --git a/packages/analytics/amplify_analytics_pinpoint/pubspec.yaml b/packag
 +version: 2.4.0
 -  amplify_core: ">=2.3.0 <2.4.0"
 +  amplify_core: ">=2.4.0 <2.5.0"
--  aws_common: ">=0.7.1 <0.8.0"
-+  aws_common: ">=0.7.2 <0.8.0"
+diff --git a/packages/analytics/amplify_analytics_pinpoint_dart/CHANGELOG.md b/packages/analytics/amplify_analytics_pinpoint_dart/CHANGELOG.md
++## 0.4.4
++
++- Minor bug fixes and improvements
++
 diff --git a/packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml b/packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml
+-version: 0.4.3
++version: 0.4.4
```

Comment:
Note to reviewers: This is an example of a bug being fixed where version bumps were not properly propagated. Since amplify_analytics_pinpoint_dart has a dependency constraint being updated, the package version needs to be updated as well.